gui/src/App.tsx

```diff
@@ -118,6 +119,10 @@ function App() {
   const dispatch = useDispatch();
   useSetup(dispatch);
 
+  useEffect(() => {
+    console.log("Is overlay? ", (window as any).isOverlayPearAI);
```

Comment:
to remove

Reply:
The console.log was already there.  I just put it into a useEffect so it would stop filling up the debugger.  I will remove it.